model_compression_toolkit/core/common/quantization/quantization_params_generation/qparams_computation.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 # ==============================================================================
 from typing import List
+from tqdm.auto import tqdm
```

Comment:
why from tqdm.auto and not from tqdm? (same as in [here ](https://github.com/sony/model_optimization/blob/69060f6821bbfed920961d807fd66b2d9046b2b5/tutorials/quick_start/keras_fw/utils.py#L18) and all other use cases in the code)

Reply:
just used to be use it that way from the days we had to explicitly select notebook vs cli. I remove it, maybe it's not required anymore.

Reply:
Still open